types/insane/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@typescript-eslint/consistent-type-definitions": "off"
```

Comment:
Q: Which line is being triggered by this?

Reply:
Any type triggers: 
```
Use an `interface` instead of a `type`.eslint[@typescript-eslint/consistent-type-definitions](https://typescript-eslint.io/rules/consistent-type-definitions)
```
I have removed the file.